sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -441,13 +442,6 @@ public void dispose() {
                 .log("Unable to emit shutdown signal.");
         }
 
-        final Mono<Void> cbsCloseOperation;
```

Comment:
For future referrence, and so that we don't add it back, it would be good to comment here on why we don't need to handle CBS node close operation.

Reply:
Sure. That's a good suggestion. 👍